habitat_baselines/common/obs_transformers.py

```diff
@@ -154,7 +154,7 @@ def __init__(
         trans_keys: The list of sensors it will try to centercrop.
         """
         super().__init__()
-        if isinstance(size, numbers.Number):
+        if isinstance(size, (int, numbers.Number)):
```

Comment:
This doesn't make any sense: https://docs.python.org/3/library/numbers.html int is a subclass.